src/main/scala/temple/Application.scala

```diff
@@ -24,9 +24,13 @@ object Application {
     val project            = ProjectBuilder.build(analyzedTemplefile, detail)
     val filteredProject    = RegenerationFilter.filter(outputDirectory, project, questionAsker)
 
-    FileUtils.outputProject(outputDirectory, filteredProject)
+    filteredProject match {
+      case Some(project) =>
+        FileUtils.outputProject(outputDirectory, project)
+        println(s"Generated project in $outputDirectory")
+      case None => println("Nothing to generate")
+    }
```

Comment:
Maybe I’m too much of a Swift dev /s, but I feel like the indentation isn’t the nicest way of handling this
```scala
val filteredProject = RegenerationFilter.filter(outputDirectory, project, questionAsker) getOrElse {
  println("Nothing to generate")
  return
}
```